libraries/AP_InternalError/AP_InternalError.cpp

```diff
@@ -125,3 +126,12 @@ void AP_memory_guard_error(uint32_t size)
         AP_HAL::panic("memory guard size=%u\n", unsigned(size));
     }
 }
+
+void AP_flash_error(uint32_t errors)
+{
+    INTERNAL_ERROR(AP_InternalError::error_t::flash);
+    if (!hal.util->get_soft_armed()) {
```

Comment:
I don't think we should panic. It would make the board not bootable if there is an error at startup. We also risk flooding the console with the error msg, so best to drop this whole section and just to internal error I think

Reply:
Ok fixed